packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.styles.ts

```diff
@@ -75,6 +85,7 @@ export const getStyles = (props: IGroupHeaderStyleProps): IGroupHeaderStyles =>
       selected && [
         classNames.isSelected,
         {
+          borderColor: palette.white,
```

Comment:
are there no "listItemSelectBorder" or something?

Reply:
I realized that I don't even need to reassign the color in here))) So I removed it completely.